cors.go

```diff
@@ -1 +1,126 @@
+// Copyright 2021 Flamego. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+// Package cors is a middleware that generates CORS headers for Flamego.
 package cors
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/flamego/flamego"
+)
+
+const anyDomain = "!*"
+
+// Options contains options for the cors.CORS middleware.
+type Options struct {
+	// Scheme may be http or https as accepted schemes or the '*' wildcard to accept any scheme. (default: "http")
+	Scheme string
```

Comment:
```suggestion
	// Scheme may be http or https as accepted schemes or the '*' wildcard to accept any scheme. Default is "http".
	Scheme string
```

Please take a look at how field docstring is written in other Options:

https://github.com/flamego/flamego/blob/eff1d929e6ce0aadc12c1493918d02bdb0d04798/render.go#L33-L42

Same for other fields in the struct.